components/http/actions/validate-webhook-auth/validate-webhook-auth.mjs

```diff
@@ -0,0 +1,95 @@
+import http from "../../http.app.mjs";
+
+export default {
+  name: "Validate Webhook Auth",
+  version: "0.0.1",
+  key: "http-validate-webhook-auth",
+  description: "Require authorization for incoming HTTP webhook requests.",
+  type: "action",
+  props: {
+    http,
+    authInput: {
+      type: "string",
+      label: "Authorization Value to Authenticate",
+      description: "Select the location of the authorization value that you'd like to check. For example, if you're looking for a Bearer token on the inbound webhook request, set this to `{{steps.trigger.event.body.headers.authorization}}`.",
+      default: "{{steps.trigger.event.headers.authorization}}",
+    },
+    authType: {
+      type: "string",
+      label: "Authorization Type",
+      description: "Select the Authorization Type for the incoming webhook.",
```

Comment:
```suggestion
      description: "Select the Authorization Type for the incoming webhook.",
```

Reply:
Changed 'Authorization Type' to 'authorization type' to follow sentence case rule in Google's Developer docs style guide